chatkit-core/src/main/kotlin/com/pusher/chatkit/model/mappers/Room.kt

```diff
@@ -0,0 +1,35 @@
+package com.pusher.chatkit.model.mappers
+
+import com.pusher.chatkit.model.network.ReadStateApiType
+import com.pusher.chatkit.model.network.RoomApiType
+import com.pusher.chatkit.model.network.RoomMembershipApiType
+import com.pusher.chatkit.rooms.Room
+
+internal fun mapToRoom(
+        room: RoomApiType,
+        memberships: RoomMembershipApiType?,
+        readState: ReadStateApiType?
+) = mapToRoom(
+        room = room,
+        memberships = memberships?.userIds.orEmpty().toSet(),
```

Comment:
`memberships` should not be optional and there should not be `orEmpty` call as we are using this mapping only for joined rooms hence this information should be always present